packages/salesforcedx-sobjects-faux-generator/package.json

```diff
@@ -34,7 +34,7 @@
   "scripts": {
     "compile": "webpack",
     "webpack": "webpack",
-    "lint": "tslint --project . -e **/src/data/minSObjects.json",
+    "lint": "tslint --project . -e **/minSObjects.json",
```

Comment:
not sure what was fixed in lerna to make this start failing post upgrade, but :fixed: